paystack/src/debug/java/co/paystack/android/TestActivity.kt

```diff
@@ -0,0 +1,11 @@
+package co.paystack.android
```

Comment:
This file is not in the test package any reason why since its a test Activity?

Reply:
It's the activity used to launch an ActivityScenario. It's here because it can be used in Instrumentation and Robolectric tests. Putting it in the `test` sourceset would mean we cannot reach it from `androidTests`